sirepo/sim_data/__init__.py

```diff
@@ -677,11 +751,28 @@ def split_jid(jid):
 def _init():
     global cfg
 
+    sirepo.job.init()
```

Comment:
It could be in util. But, it makes things tricky. Now that util defines the locks (simulation_db and sim_db_file) it matters when and how it gets initialized. sim_data doesn't know what context (server or supervisor) it is running in so it doesn't know how to initialize util.

Reply:
Potentially sim_data could be running in a pkcli. It's kind of messed up. As we decouple sirepo from flask and tornado, we'll have to normalize the init.